ci/labels/.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.cjs]
```

Comment:
Should this editorconfig also cover `.js` (run.js), `.json`, etc? Or is that covered by editorconfig files in a parent directory?